### ISSUE TITLE
Extend globals object to handle member names

### ISSUE THREAD

**Issue by paazmaya** [eslint/eslint#1171]
The `globals` object configuration could be extended to handle specific members of the given global.

For example allowing globally to access `MyStuff.doSomething`, while keeping other members invalid, the following configuration could be used:

``` js
"globals": {
  "jQuery": false,
  "MyStuff": {
    "doSomething": false
  }
}
```

This would allow catching any use of other members or possible typos for the `MyStuff` object.
`jQuery` is in the example just as a reference of how the globals are normally used.
